modules/core/shared/src/main/scala/lucuma/core/model/GuideConfig.scala

```diff
@@ -59,7 +59,7 @@ object GuideConfig {
         MountGuideOption.MountGuideOff,
         M1GuideOff,
         M2GuideOff,
-        false,
+        None,
```

Comment:
How is a `None` value interpreted?

Reply:
false, but it is not really used on observe